### DISCUSSION TITLE
Font Size in Dalamud

### DISCUSSION THREAD

**Discussion by kaegthegamer** [goatcorp/Dalamud#637] [Q&A] [answered]
Is there a way that an option to make the text of the UI and config bigger for those of us who have sight issues, or who are sight-impaired? Personally, I'm really enjoying the plugin so far, but it is taxing on my sight.

**Comment by Aireil** [comment, marked as answer]
You can scale Dalamud UI in the settings (/xlsettings), but not all plugins are made with this scaling in mind.
